File: AgriAlert/AgriAlert_IOS/AgriAlert/RegistrationRequest.swift
//
//  RegistrationRequest.swift
//  AgriAlert
//
//  Created by Fattouhi Radwa on 23/12/2024.
//

//
//  RegistrationRequest.swift
//  AgriAlert
//

import Foundation

struct RegistrationRequest: Encodable {
    let firstName: String
    let lastName: String
    let email: String
    let password: String
    let phoneNumber: String
    let location: Location
    let crops: [String]
}

struct Location: Encodable {
    let latitude: Double
    let longitude: Double
}

File: AgriAlert/AgriAlert_Mobile/app/src/main/java/ma/ensaj/agri_alert/model/RegistrationRequest.kt
package ma.ensaj.agri_alert.model

data class RegistrationRequest(
    val firstName: String,
    val lastName: String,
    val email: String,
    val password: String,
    val phoneNumber: String,
    val location: Location,
    val crops: List<String>
)

data class Location(
    val latitude: Double,
    val longitude: Double
)
